manifests/server/config.pp

```diff
@@ -35,6 +35,13 @@
     mode   => '0700',
   }
 
+  file { $mcollective::ssldir:
```

Comment:
Do you mean an if statement with its own parameter (like $ssldir_manage) or one that is a test on an existing fact/parameter (like $middleware_ssl)?

I can't see the use case where someone wouldn't want this managed except if they weren't using the ssl security provider.